### PR TITLE
Add alert when devtools info copied to clipboard

### PR DESCRIPTION
**Pull request by AmbrishRamachandiran**
image

### CODE REVIEW

**Review by awanlin — CHANGES_REQUESTED** [review]
Thanks for this @AmbrishRamachandiran, this is a nice addition! 🚀 

The build is failing as you need to regenerate the API report and commit those changes, details here if you need them: https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#api-reports

File: plugins/devtools/src/components/Content/InfoContent/InfoContent.tsx
Comment (awanlin) [inline plugins/devtools/src/components/Content/InfoContent/InfoContent.tsx:70]: You need to add back this line to fix the API report warning

**Review by awanlin — CHANGES_REQUESTED** [review]

**Review by awanlin — APPROVED** [review]
Awesome! Everything looks good to me now, thanks again @AmbrishRamachandiran 🚀 

Just need a review from the maintainers now.

File: plugins/devtools/src/components/Content/InfoContent/InfoContent.tsx
Comment (Rugvip) [inline plugins/devtools/src/components/Content/InfoContent/InfoContent.tsx:73]: Please use this instead: `import useCopyToClipboard from 'react-use/lib/useCopyToClipboard'`. It's kinda hard to get right and requires a fair bit of code, so we prefer that utility instead.

**Review by Rugvip — CHANGES_REQUESTED** [review]

File: plugins/devtools/src/components/Content/InfoContent/InfoContent.tsx
Comment (awanlin) [reply plugins/devtools/src/components/Content/InfoContent/InfoContent.tsx:73]: Oh, cool, didn't know about that! This is mostly my original code moved around so any further improvements would be great 👍

File: plugins/devtools/src/components/Content/InfoContent/InfoContent.tsx
Comment (awanlin) [inline plugins/devtools/src/components/Content/InfoContent/InfoContent.tsx:64]: Would it make sense to use the `state` like they do in the docs - https://github.com/streamich/react-use/blob/master/docs/useCopyToClipboard.md#usage - and then use the `alertApi` to log the error?

**Review by awanlin — CHANGES_REQUESTED** [review]

**Review by awanlin — APPROVED** [review]
Thanks for iterating on this @AmbrishRamachandiran, looks good to me now! 🚀 

Just need approval form a Maintainer now 👍

File: plugins/devtools/src/components/Content/InfoContent/InfoContent.tsx
Comment (Rugvip) [inline plugins/devtools/src/components/Content/InfoContent/InfoContent.tsx:78]: Could you add a test for this? I don't think this code will actually trigger an alert

**Review by Rugvip — CHANGES_REQUESTED** [review]

File: plugins/devtools/src/components/Content/InfoContent/InfoContent.tsx
Comment (AmbrishRamachandiran) [reply plugins/devtools/src/components/Content/InfoContent/InfoContent.tsx:78]: you mean need to right test case? or modify the code

Comment (Rugvip) [reply plugins/devtools/src/components/Content/InfoContent/InfoContent.tsx]: Yep it'd be best to add a test case for this to make sure it works

**Comment by AmbrishRamachandiran** [comment]
Help wanted for test case addition please happy to collaborate with someone

**Pull request closed without merging**
